etl/transform/elasticsearch.py

```diff
@@ -132,15 +132,19 @@ def validate_documents(document_tuples, validation_schemas, logger):
     logger.debug(f"Validated {document_count} documents.")
 
 
-def dump_in_json_files(source_dir, logger, documents_tuples):
+def dump_clean_in_json_files(source_dir, logger, documents_tuples):
     """
-    Consumes an iterable of document tuples
+    Consumes an iterable of document tuples and clean email
     """
     logger.debug("Saving documents to json files...")
 
     json_list = list()
     document_count = 0
     for document_header, document in documents_tuples:
+        if "contact" in document:
+            old_mail = document["contact"]["email"]
+            safe_mail = old_mail.replace('@', '_')
```

Comment:
Is `email` mandatory? What if None?

Reply:
right

Reply:
Done, a little bit more complicated than expected